src/test/scala/org/broadinstitute/hail/methods/IBDSuite.scala

```diff
@@ -31,8 +31,10 @@ class IBDSuite extends SparkSuite {
   implicit object ibdAbsoluteFuzzyComparable extends AbsoluteFuzzyComparable[IBDInfo] {
     def absoluteEq(tolerance: Double, x: IBDInfo, y: IBDInfo) = {
       def feq(x: Double, y: Double) = AbsoluteFuzzyComparable.absoluteEq(tolerance, x, y)
+      def NaNorFeq(x: Double, y: Double) =
+        x.isNaN && y.isNaN || feq(x, y)
```

Comment:
cc @cseed

In general, I'm not sure this is the right approach. I wanted some way to handle doubles no matter where they appear. I also wanted to avoid re-writing this code. This seemed like an understandable solution, but it feels a bit more complex than I wanted.

I think `D_==` is actually `RelativeFuzzyComparable` (which I should probably change to have consistent behavior wrt to NaNs).